Added unroll_twitter_thread command

DanielWils opened the pull request:
Added twitter unroll command.
Note: these changes require an update to config.json

DanielWils committed:
Added unroll_twitter_thread command

DanielWils committed:
Syntax fix

DanielWils committed:
unroll_twitter_thread improvements

michaelFarrier commented:
Looks good, just wanted to run by a few things:

- Assuming the user tries to unroll a single thread tweet, they'll get both the more than week old error & the nothing to unroll error. Can be a bit misleading as to what the actual problem is. Not really game breaking though.

- Ran into an error when I messed around with an invalid link. May want to add a check for undefined/invalid tweets on line 60. Link I used: https://twitter.com/porterrobinson/status/111111111111. Assuming most users just copy and paste the twitter links, this shouldn't be too bad of an issue, but it can happen if maybe they didn't highlight the whole link and missed a few characters.

- Is there any way we can incorporate some of the tweet details into the thread name? Like "Unrolled Twitter Thread from [twitter user]". If a user wanted to search for a particular thread in the future and many twitter unrolls have been performed then the thread archives will be flooded with "Unrolled Twitter Thread" and they'll have to go through each one to find what they are looking for.

DanielWils committed:
Thread unroller improvements

DanielWils commented:
Added invalid tweet handling & discord thread naming to include twitter handle. The individual tweet unrolling case is difficult to handle because of how twitter structures tweets but I updated the response messages to hopefully make it clearer. I also broke out our response messages to make them easier to modify in the future.

michaelFarrier reviewed (approved):
looks good
